nixos/tests/common/x11.nix

```diff
@@ -1,9 +1,14 @@
 { lib, ... }:
 
-{ services.xserver.enable = true;
+{
+  imports = [
+    ./auto.nix
```

Comment:
This is better than before but I would much rather do something like

```nix
imports = [
  (import ./common/auto-login.nix { user = "alice"; })
];
```

at each individual modules.

But it would not be idiomatic and I am not sure if it would not affect the performance negatively.

Reply:
This whole thing could really turn into a common testing lib which I think we actually might need.
But I don't think I really want to improve this more just yet, I'd like to see this in 20.03.